### PR TITLE
Define api post method. Implement CreateTask method

### PR DESCRIPTION
**Pull request by sergeykuzmich**

### CODE REVIEW

File: api.go [hunk 1]
```diff
@@ -46,7 +57,8 @@ func (a *API) Get(path string, args Arguments, target interface{}) error {
 	if res.StatusCode != 200 {
 		var body []byte
 		body, _ = ioutil.ReadAll(res.Body)
-		return errors.Errorf("HTTP request failure on %s: %s", url, string(body))
+		err := errors.New(strconv.Itoa(res.StatusCode))
+		return errors.Wrapf(err, "HTTP request failure on %s: %s", url, string(body))
```
Comment (sergeykuzmich) [inline api.go]: Weird thing happens here: there is new error wrapped once with detailed message, but when you try to check original error (http status code) it requires you to unwrap error twice. https://github.com/sergeykuzmich/harvestapp-sdk/blob/3bc79f909c8d86e571a27a770c16043347f36272/task_test.go#L27

**Comment by sergeykuzmich** [comment]
Next I want to rewrite Get and Post API methods to use single one base method due its duplications.

File: api.go (hunk 1 above)
Comment (eplatonoff) [reply api.go]: It's because Wrapf creates withStack contained withMessage contained original.

Comment (eplatonoff) [reply api.go]: So I suggest we create specific error types in future instead of decorated status codes

**Review by eplatonoff — APPROVED** [review]

**Comment by sergeykuzmich** [comment]
> Next I want to rewrite Get and Post API methods to use single one base method due its duplications.

Done.

**Pull request merged**